### PR TITLE
Modified docs to account for changes made in FLUID-4016

### PR DESCRIPTION
**Pull request by shadxx7**
Modified docs to account for https://issues.fluidproject.org/browse/FLUID-4016

### CODE REVIEW

File: src/documents/APIChangesFrom2_0To3_0.md [hunk 1]
```diff
@@ -121,3 +121,9 @@ All panels must supply a `header` selector, which will be used by the `fluid.pre
         </tr>
     </tbody>
 </table>
+
+## Component API Changes
+
+### Reorderer Component
+
+* The 'stylisticOffset' selector has been removed as it was of no use.
```
Comment (jobara) [inline src/documents/APIChangesFrom2_0To3_0.md]: I think you've used single quotes instead of back ticks around stylisticOffset. Also can you rephrase this to something like. "The `stylisticOffset` selector was no longer in use and has been removed.

**Comment by shadxx7** [comment]
I have made the change accordingly

File: src/documents/ImageReordererAPI.md [hunk 2]
```diff
@@ -237,7 +236,6 @@ selectors: {
     selectables: ".flc-reorderer-movable",
     dropTargets: ".flc-reorderer-movable",
     grabHandle:  "",
```
Comment (jobara) [inline src/documents/ImageReordererAPI.md]: There's not a trailing comma after `grabHandle: ""`

Comment (shadxx7) [reply src/documents/ImageReordererAPI.md]: oops, will fix it soon :)

**Comment by jobara** [comment]
merged at d942655fb33122a9cfa36b605bd086a00e78ca32
